api/events/api.go

```diff
@@ -2,6 +2,12 @@ package events_api
 
 import "database/sql"
 
+// ActionRepresentation struct
+type ActionRepresentation struct {
```

Comment:
The same structure is used by papercard module.  
If for the long term this stays the same, then we can move them in common service

Reply:
I will move it in common during PR2